docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+
+1. At  the beginning of the Makefile, there is a block of `DESIGN_CONFIG` variables that are commented out. These variables tell OpenROAD which design to run and on what platform. `DESIGN_CONFIG` specifically points to a `config.mk` file located in the designs directory for the respective platform. A `DESIGN_CONFIG` variable must be added for the new platform using a specific design. OpenROAD has multiple Verilog designs already made which can be used with any platform (see `OpenROAD-flow-scripts/flow/designs/src` for a list of usable designs). For example, a `DESIGN_CONFIG` variable using the `riscv32i` design on a new platform would look as follows:
+
+
+```
+DESIGN_CONFIG=./designs/MyNewPlatform/riscv32i/config.mk
+```
+
+
+The `config.mk` file will be generated later in the [Design Directory](#DesignDirectory) section of this document.
+
+
+## Platform Directory
+
+Create a directory for the new technology inside `OpenROAD-flow-scripts/flow/platforms` to contain the necessary files for the OpenROAD flow.
+
+
+```
+$ mkdir OpenROAD-flow-scripts/flow/platforms/MyNewPlatform
+```
+
+
+
+## Design Directory
+
+The design directory contains the configuration files for all the designs of a specific platform. Create a directory for the new platform in OpenROAD-flow-scripts/flow/designs to contain the relevant files and directories for all the designs for the flow in that specific platform. Each design requires its own `config.mk` and `constraints.sdc` files. 
+
+Follow the steps below to create the necessary directories and files. **NOTE: riscv32i is just an example and not a required name.**:
+
+
+```
+$ mkdir -p OpenROAD-flow-scripts/flow/designs/MyNewPlatform/riscv32i
+$ touch OpenROAD-flow-scripts/flow/designs/MyNewPlatform/riscv32i/config.mk
+$ touch OpenROAD-flow-scripts/flow/designs/MyNewPlatform/riscv32i/constraints.sdc
+```
+
+
+This creates two directories MyNewPlatform and riscv32i and two empty files `config.mk` and `constraints.sdc` in OpenROAD-flow-scripts/flow/designs/MyNewPlatform/riscv32i. 
+
+
+## Platform Configuration
+
+This section describes the necessary files in the platform directory needed for the OpenROAD flow. Specifically the `config.mk` file in the platform directory has all of the configuration variables that the flow uses. Refer to the OpenROAD-flow-scripts documentation for a full list of configuration variables that can be set.
+
+For an example of a platform config.mk file, refer to `OpenROAD-flow-scripts/flow/platforms/sky130hd/config.mk.`
+
+
+## Design Configuration
+
+This section describes files in the design directory.
+
+
+## config.mk
+
+The `config.mk` file describes design-specific variables.
+
+For Example:
+
+
+```
+    DESIGN_NAME
+    PLATFORM
+    VERILOG_FILES
+    SDC_FILE
+    CORE_UTILIZATION
+    CORE_ASPECT_RATIO
+    CORE_MARGIN
+    PLACE_DENSITY
+```
+
+Alternatively, in place of `CORE_UTILIZATION` and `CORE_ASPECT_RATIO`, `DIE_AREA` and `CORE_AREA` can be specified instead.
```

Comment:
`DIE_AREA` and `CORE_AREA` can be specified instead of `CORE_UTILIZATION`, `CORE_ASPECT_RATIO`, and `CORE_MARGIN`.

(reworded to avoid confusing abutting lists)

Reply:
Reworded